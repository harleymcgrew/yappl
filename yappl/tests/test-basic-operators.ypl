~print_line (-2.5e-6 + 1.0);
~print_line (-2 + 1000);
~print_line (8 * 5);
~print_line (90 -15);
~print_line (1000/10);
~print_line (2 * (-2) + 102 - 6 / 3);
~print_line (10 % 5)
