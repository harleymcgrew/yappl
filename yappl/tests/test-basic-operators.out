0.9999975
998
4
75
100
96
0